src/openrct2/ride/Ride.h

```diff
@@ -298,6 +298,7 @@ struct Ride
     uint16_t totalAirTime{};
     StationIndex currentTestStation{ StationIndex::GetNull() };
     uint8_t numCircuits{};
+    bool musicFromAllStations{ true }; // Play music from all stations if true, only main if false
```

Comment:
This could probably be part of lifecycleFlags. Import code will have to shift to set it on parkfiles between this version and when the feature was introduced.

Reply:
The "lifecycle" flags are the main flags field for the ride, not just relating to the ride's operation. The music flag is in there, for example.

If it's going to be a per-station option, it should go in the RideStation struct. I don't see how 255 stations can be individually controlled from just a uint8_t.

Reply:
No, I meant a tristate:
1. Play music on first station only
2. Play same music on all the stations
3. Play individual music on each station

Now that I spelled it out, I see that 1 is actually a special case of 3, where only one station has the music set… But the UI for that would have to thought out to make it easy to manage.

Still, I stick with my choice of member field until others chime in.